src/inputosmpbf.cpp

```diff
@@ -56,7 +56,7 @@ struct field_t
 
 struct string_table_t
 {
-    std::vector<uint32_t> st_index;
+    std::vector<std::size_t> st_index;
```

Comment:
strange that this actually solves it, we suspect it may be some instrumented vs non instrumented code issue, but we can investigate separately

in any case, if this solves it for now, let's go